5-update-&-rollback.md

```diff
@@ -38,7 +38,7 @@ git commit -m "Added buggy alert"
 Build the container and deploy the latest revision:
 ```bash
 nscale system compile sudc development
-nscale container build sudc web
+nscale container build sudc web development
 nscale revision deploy sudc latest development
 ```
```

Comment:
A target can be abbreviated, but ok for consistency!
Il giorno gio 26 feb 2015 alle 12:15 Dean McDonnell <
notifications@github.com> ha scritto:

> In 5-update-&-rollback.md
> <https://github.com/nearform/nscale-workshop/pull/39#discussion_r25419658>
> :
>
> > @@ -38,7 +38,7 @@ git commit -m "Added buggy alert"
> >  Build the container and deploy the latest revision:
> >  ```bash
> >  nscale system compile sudc development
> > -nscale container build sudc web
> > +nscale container build sudc web development
> >  nscale revision deploy sudc latest development
> >  ```
>
> Should the command be dev over development here? it seems the docs have
> dev in some places, development in others. Unless there is a reason we
> should perhaps stick to one convention for docs? @darragh-hayes
> <https://github.com/darragh-hayes> can you check this? @mcollina
> <https://github.com/mcollina> can you confirm?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nearform/nscale-workshop/pull/39/files#r25419658>.
>